pykafka/simpleconsumer.py

```diff
@@ -408,37 +410,50 @@ def _handle_success(parts):
             to_retry.extend(parts_by_error.get(NotCoordinatorForConsumer.ERROR_CODE, []))
             reqs = [p.build_offset_fetch_request() for p, _ in to_retry]
 
-    def _reset_offsets(self, partitions=None):
-        """Reset offsets after an error
+    def reset_offsets(self, partitions=None):
+        """Reset offsets for the specified partitions
 
         Issue an OffsetRequest for each partition and set the appropriate
-        returned offset in the OwnedPartition per self._auto_offset_reset
+        returned offset in the OwnedPartition
 
-        :param partitions: the partitions for which to reset offsets
+        :param partitions: (`partition`, `offset`) pairs to reset
+            where `partition` is the partition for which to reset the offset
+            and `offset` is the new offset the partition should have
         :type partitions: Iterable of
-            :class:`pykafka.simpleconsumer.OwnedPartition`
+            (:class:`pykafka.simpleconsumer.OwnedPartition`, int)
```

Comment:
Could this become "Iterable of (int, int)"  - so that is, (partition_id, offset) - instead? Otherwise a user would have to go through `self._partitions_by_id` to obtain instances of `OwnedPartition` first before they'd call this, I think.